.github/workflows/build.yml

```
@@ -0,0 +1,25 @@
+name: Build
+
+on:
+  push:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Your comment was fair, let's set the actual value
```suggestion
    runs-on: ubuntu-20.04
```